src/site/layouts/va_form.drupal.liquid

```diff
@@ -0,0 +1,70 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
```

Comment:
Are we sure we don't have a `main` tag as a parent?

Reply:
I don't think so. I followed the `page.drupal.liquid` template as an example